mesheryctl/cmd/cleanup.go

```diff
@@ -22,8 +22,8 @@ import (
 // cleanupCmd represents the cleanup command
 var cleanupCmd = &cobra.Command{
 	Use:   "cleanup",
-	Short: "Cleanup Meshery's configuration",
-	Long:  `Reset Meshery to it's default configuration.`,
+	Short: "Cleaning up Meshery's configuration",
```

Comment:
Aren't we moving to change this command to reset ? @leecalcote

Reply:
@thakursaurabh1998 yes, that's right. 👍